sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1133,9 +1124,22 @@ void foo(queue q /* queue in recording mode */ ) {
 }
 ----
 
-The copy will not occur if the buffer is accessed inside the submitted command
-group function with an accessor with `access_mode::write` or the `no_init`
-property.
+In this example, the implementation extends the lifetime of the buffer because
+it is used in the recorded graph. Because the buffer uses the host memory data,
+the implementation also makes an internal copy of that host data. As illustrated
+above, that host memory might go out of scope before the recorded graph goes out
+of scope.
+
+The default behavior is to always copy the host data in a case like this, but
+this is not necessary if the user knows that the lifetime of the host data
+outlives the lifetime of the recorded graph. If the user knows this is the
+case, he or she may use the graph::no_host_copy property to avoid the internal
+copy. Users can opt-out of this behavior by passing the property `graph::no_host_copy`
```

Comment:
It feels like we're defining `graph::no_host_copy` again in this sentence after we just defined it in the last sentence, we could probably just remove it.

> ~~Users can opt-out of this behavior by passing the property `graph::no_host_copy` to `command_graph::begin_recording()` or the constructor of the `command_graph` the commands will be recorded to~~